lib/vmdb/global_methods.rb

```diff
@@ -36,31 +36,23 @@ def column_type(model, column)
     end
 
     # Had to add timezone methods here, they are being called from models
-    def get_timezone_abbr(typ="server")
-      # return timezone abbreviation
-      if typ == "server"
-        tz = MiqServer.my_server.get_config("vmdb").config.fetch_path(:server, :timezone)
-        tz = ActiveSupport::TimeZone::MAPPING[tz.blank? ? "UTC" : tz]
+    # return timezone abbreviation
+    def get_timezone_abbr(user = nil)
+      if user.nil?
+        tz = get_timezone_for_userid(nil)
         time = Time.now
       else
         tz = Time.zone
```

Comment:
I find this odd that we use `time.zone` for the current user here but use something else in the next 2 methods. Gotta love refactoring when it reveals stuff like this.

@h-kataria @eclarizio maybe we can circle back to understand this distinction. Many other places in the code copy and paste this logic while other places assume the current user's get_timezone